tests/run/extra_patma.pyx

```diff
@@ -16,3 +15,64 @@ def test_is_null():
     return  # disabled - currently just a parser test
```

Comment:
This should also be removed to enable the test. I'll push it in the next round of updates (doesn't seem worth rerunning everything for that)